src/components/TaskList/TaskList.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import React,{useState} from 'react'
+import styles from '@/components/TodoList/TodoList.module.css'
+
+function TaskList({keyId,value,editListHandler,removeListHandler}) {
+
+    const [editTaskInput,setEditTaskInput ] = useState(value)
+    const [checkBox,setCheckBox] = useState(false);
```

Comment:
This is a very weird naming convention. Instead of this, you can use isCompleted.